records_versioning/services.py

```
@@ -0,0 +1,154 @@
+_thread_state = threading.local()
+
+IGNORE_FIELDS = {"id", "pk", "created_at", "updated_at"}
+
+RESOURCE_MAP = {
+    'Patient': ('patients_core', 'Patient'),
+    'Encounter': ('diab_encounters', 'Encounter'),
+    'LabResult': ('diab_labs', 'LabResult'),
+    'MedicationOrder': ('diab_medications', 'MedicationOrder'),
+}
+
+def _compute_snapshot(instance):
```

Ruff (0.12.2)

Missing return type annotation for private function `_compute_snapshot`

(ANN202)